lib/model/query/auth.js

```diff
@@ -38,13 +38,17 @@ const can = (actor, verbs, actee) => ({ oneFirst }) => {
   }
 
   return oneFirst(sql`
-${_impliedActees(acteeId)}
-select count(*) from assignments
-inner join implied on implied.id=assignments."acteeId"
-inner join (select id from roles where ${_verbClause(verbsSet)}) as role on role.id=assignments."roleId"
-where "actorId"=${actor.id}
-limit 1`)
```

Comment:
I know the `limit 1` didn't do much before, because the `count(*)` would always be one row. But now I'm wondering, would adding `LIMIT 1` to the new `SELECT 1` subquery improve its performance? We just want to know if any assignment exists, so we don't need to retrieve all assignments. For projects with many forms, app users can have many assignments. That might be outside the scope of this PR, but I thought I'd throw out the idea.

Reply:
I don't think it's necessary per the postgres docs:

> The subquery will generally only be executed long enough to determine whether at least one row is returned, not all the way to completion.
> - _https://www.postgresql.org/docs/14/functions-subquery.html#FUNCTIONS-SUBQUERY-EXISTS_